<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>h2s</class>
 <widget class="QWidget" name="h2s">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>316</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>hex2SAE</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#h2s{
border-image: url(:/image/bg.jpg);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="input">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#input{
color: rgba(0, 255,255, 250);background-color: rgba(0,0,255,100);border:none;
}</string>
           </property>
           <property name="text">
            <string>hex File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="output">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#output{
color: rgba(0, 255,255, 250);background-color: rgba(0,0,255,100);border:none;
}</string>
           </property>
           <property name="text">
            <string>SAE File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_input">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit#lineEdit_input{
background-color:rgba(255, 255, 255, 150)
}</string>
           </property>
           <property name="maxLength">
            <number>32769</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_output">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit#lineEdit_output{
background-color:rgba(255, 255, 255, 150)
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="hex2SAE">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#hex2SAE{
color: rgba(0, 255,255, 250);background-color: rgba(0,0,255,100);border:none;
}</string>
         </property>
         <property name="text">
          <string>hex2SAE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SAE2hex">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#SAE2hex{
color: rgba(0, 255,255, 250);background-color: rgba(0,0,255,100);border:none;
}</string>
         </property>
         <property name="text">
          <string>SAE2hex</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="Return">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
